Description: New line skip in weather.pl.
 Previously, there were 12 lines between the "START BODY OF TEXT HERE" comment
 and the time in the html file downloaded from weather.noaa.gov for processing
 weather information.  Now, however, the number of lines has increased to 21,
 and weather.pl was not able to correctly parse the weather information.
 .
 This patch fixes this issue.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-24

--- a/Src/weather.pl
+++ b/Src/weather.pl
@@ -79,7 +79,7 @@
 
     $i = 0;
 
-    while ( $i != 12  and  not eof DATA )
+    while ( $i != 21  and  not eof DATA )
     {
 	$line = <DATA>;
 	$i++;
